gcloud/storage/test_batch.py

```diff
@@ -362,6 +354,16 @@ def test_finish_nonempty_non_multipart_response(self):
         batch._requests.append(('DELETE', URL, {}, None))
         self.assertRaises(ValueError, batch.finish)
 
+    def test_current(self):
+        from gcloud.storage.batch import _BATCHES
+        klass = self._getTargetClass()
+        batch_top = object()
+        self.assertEqual(list(_BATCHES), [])
+        _BATCHES.push(batch_top)
+        self.assertTrue(klass.current() is batch_top)
+        _BATCHES.pop()
+        self.assertEqual(list(_BATCHES), [])
```

Comment:
Was put in to keep test coverage at 100% (some code path that used to call `.current()` is getting axed here). Ends up being irrelevant in #990 when the local stack is used on the client.